### DISCUSSION TITLE
Display NuGet package for a give type

### DISCUSSION THREAD

**Discussion by pepone** [dotnet/docfx#8716] [Q&A] [answered]
Any ideas on how to show the NuGet package for each type, similar to what Microsoft does?

<img width="607" alt="nuget-package" src="https://user-images.githubusercontent.com/254604/235908348-dfac8d6e-4353-481d-a591-49bc48557368.png">

**Comment by yufeih** [comment, marked as answer]
<https://learn.microsoft.com> did it through an external config [here](https://raw.githubusercontent.com/dotnet/dotnet-api-docs/main/xml/PackageInformation/dotnet-plat-ext-6.0.json). This is not yet supported by docfx.
